neo/Prompt/CommandBase.py

```diff
@@ -66,8 +66,11 @@ def command_desc(self):
         pass
 
     # Raise KeyError exception if the command does not exist
-    def execute_sub_command(self, id, arguments):
-        self.__sub_commands[id].execute(arguments)
+    def execute_sub_command(self, id, arguments=None):
+        if arguments is not None:
```

Comment:
In my opinion, the original signature `execute(self, arguments)` should be respected. None could be given as arguments for commands like `CommandWalletClose`.